api/src/main/java/io/opentelemetry/common/AttributeKey.java

```diff
@@ -25,4 +26,44 @@
   /** Returns the type of attribute for this key. Useful for building switch statements. */
   @Nonnull
   AttributeType getType();
+
+  /** Create a new AttributeKey for String valued attributes. */
```

Comment:
Let's change these to `Returns a` while here.

Reply:
Done